### ISSUE TITLE
AndroidManifest.xml conflict

### ISSUE THREAD

**Issue by wangbo4020** [wu9007/qrcode_scanner#103]
Could move these attributes and ic_laucnher.png to Example app?
Remove from here, because here is library?
Usually these attributes define in App.
![image](https://user-images.githubusercontent.com/5559403/113853342-23876680-97d0-11eb-80d7-1b9afc58c579.png)

**Comment by lkrjangid1** [comment]
put user permission code like https://github.com/leyan95/qrcode_scanner/blob/master/example/android/app/src/main/AndroidManifest.xml

**Comment by wu9007** [comment]
@wangbo4020 yes — a library should not ship `application@icon` or `ic_launcher.png`. Screenshot was correct.

0.4.0 plugin manifest has no `android:icon`; launcher mipmaps are not in the plugin resources. Host owns the icon. Closing as completed.
